depends/Makefile

```
@@ -133,9 +133,10 @@ endef
 
 define check_or_remove_sources
   mkdir -p $($(package)_source_dir); cd $($(package)_source_dir); \
-  $(build_SHA256SUM) -c $($(package)_fetched) >/dev/null 2>/dev/null || \
-    ( if test -f $($(package)_all_sources); then echo "Checksum missing or mismatched for $(package) source. Forcing re-download."; fi; \
-      rm -f $($(package)_all_sources) $($(1)_fetched))
+  ( $(build_SHA256SUM) --check --quiet --ignore-missing $($(package)_fetched) && \
+    echo $($(package)_all_sources) | xargs -n 1 test -f \
+  ) || \
+    rm -f $($(package)_all_sources) $($(1)_fetched)
```

```suggestion
  $(build_SHA256SUM) -c $($(package)_fetched) >/dev/null 2>/dev/null || \
    ( ( echo $($(package)_all_sources) | xargs -n 1 test -f ) || echo "Checksum missing or mismatched for $(package) source. Forcing re-download."; \
      rm -f $($(package)_all_sources) $($(1)_fetched))
```
This is untested!

(Squash this so that it is only changing line 137.)

---

Thanks, I agree this is a more conservative change that just focuses on the bugfix. I'll try it out.